GalaxyZooTouchTable/GalaxyZooTouchTable/Behaviors/DisableOpacityMaskOnEndScroll.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Interactivity;
+using System.Windows.Media;
+
+namespace GalaxyZooTouchTable.Behaviors
+{
+    public class DisableOpacityMaskOnEndScroll : Behavior<ScrollViewer>
+    {
+        private LinearGradientBrush InitialOpacityMask { get; set; }
+
+        protected override void OnAttached()
+        {
+            base.OnAttached();
+
+            AssociatedObject.ScrollChanged += AssociatedObject_ScrollChanged;
+            SetOpacityMask();
+        }
+
+        private void SetOpacityMask()
+        {
+            var gradientCollection = new GradientStopCollection();
+            var gradientStop1 = new GradientStop(Color.FromRgb(0, 0, 0), 0);
+            var gradientStop2 = new GradientStop(Color.FromArgb(0, 0, 0, 0), 1);
```

Comment:
Does `Color` ship with constants like `transparent` and `black`? It may be better to use those here instead of defining the colors with rgba values